### PR TITLE
index changes

### PR DESCRIPTION
**Pull request by gabrielgiri**
guys please review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -59,7 +59,7 @@
         <div class="row">
           <div class="col-lg-10 mx-auto">
             <h1 class="text-uppercase">
-              <strong>Your Favorite Source of Free Bootstrap Themes</strong>
+              <strong>GIT FUNDAMENTALS</strong>
```
Comment (cintiahet) [inline index.html]: Wrong style title

**Review by cintiahet — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (gabrielgiri) [reply index.html]: done, please review it again

**Review by cintiahet — APPROVED** [review]

**Review by effuranga — CHANGES_REQUESTED** [review]
Wrong violet hue. Should be lighter

**Comment by gabrielgiri** [comment]
done

**Pull request merged**